lips/lip-1.mdx

```diff
@@ -498,19 +452,106 @@ Valid values are:
 * `ready_for_settlement` - Transaction is ready for settlement according to this actor (i.e. the required signatures/KYC data have been provided)
-* `settled` - Payment has been settled on chain and funds delivered to the subaddress
 * `abort` - Indicates the actor wishes to abort this payment, instead of settling it.
-* `pending_review` - Payment is pending review.
 * `soft_match` - Actor's KYC data resulted in a soft-match. The VASP associated with this actor will need to determine what next steps are required, and if it decides to send additional KYC information to clear the soft-match does so via the KYCObject field of additional_kyc_data. If not sent within SLA window, this transaction will be aborted.
 
-**Valid Status Transitions**. Each side of the transaction is only allowed to mutate their own status (sender or receiver), and upon payment creation may only set the status of the other party to `none`. Subsequently, each party may only modify their own status to a higher or equal status in the order `none`, (`needs_kyc_data`, `needs_recipient_signature`, `pending_review`, `soft_match`), (`ready_for_settlement`, `abort`).
 
-A status of `abort` on _either_ side is terminal and the payment must not be changed subsequently.
-A status of `ready_for_settlement` on _both_ sides is also terminal, and indicates that the payment should be settled on-chain and not modified further. It is therefore safe for a VASP sending funds to initiate an On-Chain payment to settle an Off-chain payment after it observed the other party setting their status to `ready_for_settlement` and it is also willing to go past this state.
+# PaymentObject Protocol Command Sequence
+
+**PaymentObject State.** The state of a `PaymentObject` is used to determine which VASP is expected to issue a command next, and what information is expected to be included in the command to progress the payment. The state is determined by the tuple of the status of the Sender and Receiver Actors and two flags `Sx` and `Rx`. The exact fields in the payment object for Sender and Receiver actor status are `sender.status.status` and `receiver.status.status`. The flag `Sx` and `Rx` are set when the Sender or the Receiver has included `additional_kyc_data` in the actor field (`Sender.kyc_data.additional_kyc_data` and `Receiver.kyc_data.additional_kyc_data` respectively).
+
+The states (Sender Status, Receiver Status, Sx Flag, Rx Flag) are:
+
+Basic KYC exchange flow
+
+* SINIT: (`need_kyc_data`, `none`, `*`, `*`)
+* RSEND: (`need_kyc_data`, `ready_for_settlement`, `*`, `*`)
+* RABORT: (`need_kyc_data`, `abort`)
+* SABORT: (`abort`, `ready_for_settlement`, `*`, `*`)
+* READY: (`ready_for_settlement`, `ready_for_settlement`, `*`, `*`)
+
+Soft-match disambiguation states:
+
+* RSOFT (`need_kyc`, `soft_match`, `_`, `_`)
+* SSOFTSEND (`need_kyc`, `soft_match`, `Sx`, `_`)
+* SSOFT (`soft_match`, `ready_for_settlement`, `*`, `_`)
+* RSOFTSEND (`soft_match`, `ready_for_settlement`, `*`, `Rx`)
```

Comment:
is receiver allowed to abort when sender send soft match?
 (soft_match, abort)

similarly, when receiver send soft_match, could sender abort?